tools/shared/src/main/scala/org/scalajs/core/tools/linker/standard/OutputModeOps.scala

```diff
@@ -0,0 +1,33 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2017, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.core.tools.linker.standard
+
+import org.scalajs.core.tools.linker._
+
+/** Import the contents of this object to enable standard `OutputMode`
+ *  selection.
+ */
+object OutputModeOps {
+  type OutputMode = org.scalajs.core.tools.linker.backend.OutputMode
+
+  val OutputMode: org.scalajs.core.tools.linker.backend.OutputMode.type =
+    org.scalajs.core.tools.linker.backend.OutputMode
+
+  implicit class StandardLinkerConfigOutputModeOps(
+      val __self: StandardLinker.Config) extends AnyVal {
+
+    import StandardLinker.Config
+
+    /** Standard output mode. */
+    def outputMode: OutputMode = __self.outputMode
+
+    def withOutputMode(outputMode: OutputMode): Config =
+      __self.withOutputMode(outputMode)
+  }
+}
```

Comment:
Me too, actually. But at the same time, in this version one can only
```scala
import org.scalajs.core.tools.linker.standard.OutputModeOps._
```
and, if they don't import anything else in `standard._`, at least they know that their code will keep working regardless of what happens to the rest of `standard`.

But I'm not convinced myself. Maybe it's better in the package object anyway.